src/components/Verify.js

```diff
@@ -30,76 +30,84 @@ import FilterTop from './FilterTop';
 import CaptureDetailDialog from './CaptureDetailDialog';
 import OptimizedImage from './OptimizedImage';
 import CaptureDetailTooltip from './CaptureDetailTooltip';
-import { selectedHighlightColor, documentTitle } from '../common/variables.js';
-import { countToLocaleString } from '../common/numbers';
-import { VerifyContext } from '../context/VerifyContext';
-import { SpeciesContext } from '../context/SpeciesContext';
-import { TagsContext } from '../context/TagsContext';
-import { CaptureDetailProvider } from '../context/CaptureDetailContext';
-
-const log = require('loglevel').getLogger('../components/Verify');
+import { documentTitle } from 'common/variables';
+import { countToLocaleString } from 'common/numbers';
+import { VerifyContext } from 'context/VerifyContext';
+import { SpeciesContext } from 'context/SpeciesContext';
+import { TagsContext } from 'context/TagsContext';
+import { CaptureDetailProvider } from 'context/CaptureDetailContext';
+
+const log = require('loglevel').getLogger('components/Verify');
 const EMPTY_ARRAY = new Array(16).fill();
 const SIDE_PANEL_WIDTH = 315;
 
 const useStyles = makeStyles((theme) => ({
   wrapper: {
-    padding: theme.spacing(2, 8, 4, 8),
-  },
-  cardImg: {
-    width: '100%',
-    height: 'auto',
-  },
-  cardTitle: {
-    color: '#f00',
+    padding: theme.spacing(2, 8),
   },
   card: {
+    position: 'relative',
+    display: 'grid',
+    padding: 0,
     cursor: 'pointer',
+    borderRadius: '16px',
     '&:hover $cardMedia': {
       transform: 'scale(1.04)',
     },
+    '&:hover $cardActions': {
+      opacity: 1,
+      transition: theme.transitions.create('opacity', {
```

Comment:
The transition should go on `cardActions` directly. At the moment, the transition doesn't run when removing the hover.